.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: php
+
+sudo: false
+
+env:
+  global:
+    - GITHUB_TOKEN=foo
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 7
+    - php: hhvm
+ allow_failures:
+    - php: hhvm
```

Comment:
I actually went ahead and fixed the tests that were breaking `hhvm` over in bf226b4062c0fb5b38fedaa7f2ae8b8d5dbabd43